### PR TITLE
Update for Brazilian Portuguese

### PR DESCRIPTION
**Pull request by katananja**
- All translation updated to follow GNOME Translation Rules for Brazilian Portuguese, this are the same rules followed by other Linux Distributions, Manuals, etc. Now we can avoid translation conflicts.
http://br.gnome.org/GNOMEBR/ErrosFrequentes
http://br.gnome.org/GNOMEBR/GuiaDoTradutor

- Brazilian Portuguese REQUIRE context, it doesn't matter what a word say, we need to see what the program is doing instead of what the word/phrase is saying to construct the phrase correctly.

- That ROM AUDITION functions means nothing in Brazilian Portuguese, so the translation was adapted to "Condição", this reflects better if the ROM is GOOD/BAD, etc.
- OK has different meanings in Brazilian Portuguese, if you ask someone if the food was good and you reply "OK", it means that the food  are between good and bad or more or less. Since "Ok" doesn't reflect if the ROM is good or not, the translation was changed to "BOA" witch means "GOOD" following Brazilian Portuguese concordance rules "Estado dA ROM" this "A" tell us to use "boA" (female) instead of "BOM"(male).
- Some parts of the translation was also adapted to fit the screen or the space the phrase is at.
- To our language, the correct translation to plugin is plug-in.
- A few more mistakes were fixed like Enter (to type in) = Digite not "Entre" (to get inside of) or Search = Pesquisa not "Busca", etc.

### CODE REVIEW

File: language/Portuguese_Brazil/strings.po [hunk 1]
```diff
@@ -392,23 +395,23 @@ msgstr "Cores"
...
+msgstr "Idioma"
 
 #: src/frontend/mame/ui/custui.cpp:161
 msgid "Show side panels"
 msgstr "Mostrar painéis laterais"
 
 #: src/frontend/mame/ui/custui.cpp:173
 msgid "Custom UI Settings"
-msgstr "Configurações Customizadas do UI"
+msgstr "Configurações Personalizadas da IU"
```
Comment (felipesanches) [inline language/Portuguese_Brazil/strings.po]: **IU**  meaning user interface (interface de usuário) sounds pretty weird. Maybe we should use the word "Interface", but that would result in a pretty long string: "Configurações Personalizadas da Interface". For that reason, a shorter, but nice, alternative could be "Personalizações da Interface", roughly meaning "UI customizations"

File: language/Portuguese_Brazil/strings.po [hunk 2]
```diff
@@ -392,23 +395,23 @@ msgstr "Cores"
...
 
 #: src/frontend/mame/ui/custui.cpp:223
 msgid "default"
 msgstr "padrão"
 
 #: src/frontend/mame/ui/custui.cpp:331
 msgid "UI Font"
-msgstr "Fonte do UI"
+msgstr "Fonte da IU"
```
Comment (felipesanches) [inline language/Portuguese_Brazil/strings.po]: Similar issue here with the uncommon **IU** usage.

You know... in the early 80s people in Brazil used to translate **CPU** to **UCP** (Unidade Central de Processamento). But nobody does that anymore! Tech-lingo broadly adopts acronyms in its original english form nowadays (such as CPU and UI or even GUI).

Comment (felipesanches) [reply language/Portuguese_Brazil/strings.po]: here I'd suggest saying something like "Tipografia da Interface de Usuário". We may consider using just "Interface" in places where "Interface de Usuário" results in too long strings.

Comment (felipesanches) [reply language/Portuguese_Brazil/strings.po]: Even "Fonte Tipográfica" (not mentioning the UI at all) would work well in this case, though. And could be a good / effective short string to be used here.

File: language/Portuguese_Brazil/strings.po [hunk 3]
```diff
@@ -392,23 +395,23 @@ msgstr "Cores"
...
 
 #: src/frontend/mame/ui/custui.cpp:336
 msgid "Bold"
```
Comment (felipesanches) [inline language/Portuguese_Brazil/strings.po]: Brazilian computer users are used to the term "Bold" refering to darker than "Regular" font weigths. But if we want to use a proper portuguese word we could translate this to "Negrito". Keeping it as "Bold" would do no harm, though.

**Comment by felipesanches** [comment]
@cuavas I did review some of the translation strings and provided some feedback as in-line comments. But I have not reviewed it all (I don't have time for that right now). So please do not understand my lack of further comments as endorsing the quality of the rest to this patch.

**Comment by katananja** [comment]
Well, most of what is there as stated before, was adaptations done to follow @cuavas standards, others to fit the screen. To me, "Custom UI Settings" could be "Personalização da Interface" I don't mind the UI or IU, but at the previous patch @cuavas complained about the missing UI or IU. "UI Font" could be "Configuração Tipográfica" or "Tipografia de tela/interface". But as I did in the previous patch, if the translation is too radical @cuavas doesn't accept it.

Same for Bold = Negrito.
I mean, as I've stated before, I'm here to help... But why fix it if @cuavas doesn't even considered? It's a hard work to translate all this, all the things you have to go trough to do it and it was never considered, 191 was pushed and all this work for nothing.

I really don't know what @cuavas want and why he is doing this.

**Comment by Ashura-X** [comment]
Yeah translating is very pain for sure and I know it very well...
Only got to remember again this thread maybe because Cuavas would be
forgotten it !
Anyway we are here to help one to other to bring a descent pt br
translation!
Hope Cuavas can apply this patch for next mame on those weeks!

Em sex, 27 de out de 2017 às 19:46, Wellington Uemura <
notifications@github.com> escreveu:

**Comment by katananja** [comment]
Me too @Ashura-X !
If there are something wrong with it, any one can patch it.

**Comment by ismaell** [comment]
> OK has different meanings in Brazilian Portuguese, if you ask someone if the food was good and you reply "OK", it means that the food are between good and bad or more or less. Since "Ok" doesn't reflect if the ROM is good or not, the translation was changed to "BOA" witch means "GOOD" following Brazilian Portuguese concordance rules "Estado dA ROM" this "A" tell us to use "boA" (female) instead of "BOM"(male).

It seems to me that the last statement is wrong, as the response ("Bom"/"Boa") must be in concordance with the subject ("[o] Estado") instead of the object ("a ROM"), so it must be masculine. Am I wrong?

**Pull request merged**